core/objects.go

```diff
@@ -445,6 +445,12 @@ type Authorization struct {
 	// lifetime + certificate lifetime.
 	Expires *time.Time `json:"expires,omitempty" db:"expires"`
 
+	// The date on which this authorization was created. Will be nil if the
+	// authorization was created before this field was added.
+	//
+	// TODO(riking): Change json tag to "-" after switching SA to grpc
```

Comment:
Should the nick on this `TODO` be updated to whoever we expect might manage this change post-gRPC migration?